tests/test_importers_multi.py

```
@@ -0,0 +1,305 @@
+            zf.write(file, file.relative_to(export_dir).as_posix())
+
+    out_dir = tmp_path / "out_claude_zip"
+    results = import_claude_export(
+        zip_path,
+        output_dir=out_dir,
+        collapse_threshold=10,
+        html=False,
+        html_theme="light",
+    )
+
+    assert results and results[0].attachments_dir is not None
```

Ruff (0.15.9)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)